.gitlab/junit_upload/junit_upload.yml

```diff
@@ -10,6 +10,8 @@ unit_tests_arm64_junit_upload:
   needs:
     - tests_deb-arm64-py3
     - tests_rpm-arm64-py3
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why do we need to wait on those since we're just uploading Windows tests?

Reply:
This step uploads for all tests that cannot upload using the standalone binary. so it includes arm (because the binary has not been built on arm) and windows because it's currently not working